### PR TITLE
add jenkins.yaml

### PR DESCRIPTION
**Pull request by chlgit**
This repo has been mirrored here https://sourcegraph.yelpcorp.com/mirrors/Yelp/paasta-tools-go. Hopefully, I don't miss anything.

### CODE REVIEW

File: jenkins.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+# See http://y/jenkinsfile for more information
+# about customizing your Jenkins pipeline.
+type: debian
+host_type: xenial
+overrides:
+  debian:
+    platforms: [xenial]
```
Comment (solarkennedy) [inline jenkins.yaml]: You need trusty, bionic too

File: yelp_package/trusty/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+RUN apt-get -q update && \
+    DEBIAN_FRONTEND=noninteractive apt-get -q install -y --no-install-recommends \
+        ca-certificates \
+        git \
+        wget \
+        ruby \ 
+        ruby-dev \ 
+        rubygems \
+        build-essential \
+        sudo \
```
Comment (solarkennedy) [inline yelp_package/trusty/Dockerfile]: why do we need sudo?

File: Makefile [hunk 3]
```diff
@@ -36,8 +37,9 @@ deb_%: clean docker_build_%
 			--deb-dist $* --deb-priority optional \
 			--name paasta-tools-go --package dist \
 			--description "CLI tools for PaaSTA in Go" \
+			--package dist/$* \
 			bin=/usr/ && \
-		chown -R $(UID):$(GID) bin dist \
+		sudo chown -R $(UID):$(GID) bin dist \
```
Comment (solarkennedy) [inline Makefile]: we should not need this.

We should not need to chown nor should we need sudo

Comment (chlgit) [inline Makefile]: @solarkennedy this is the place we need sudo.

Comment (chlgit) [reply Makefile]: Hmm.....it's owned by root so far. I need to figure out why.

**Pull request merged**